src/Forms/ValidationErrorMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {
+  ReactNode, FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { BaseTheme } from '../Theme';
+
+export interface ValidationErrorMessageProps {
+  /** Text or components to be displayed */
+  children: ReactNode;
+}
+
+const StyledValidationErrorMessage = styled.span`
+  color: ${({ theme }): string => (theme.color.text.negative)};
+  font-size: 0.75em;
```

Comment:
I think we should actually have this value defined as part of the `MarkOneTheme`, so we can enforce some consistency in sizing across various error components. We do have a section of `font` objects that contain a `family`, `size`, and `weight`, so maybe we should add an `error` field in there that includes this size.

Reply:
That's a good idea! Added through 3907f10